hathor/transaction/storage/transaction_storage.py

```diff
@@ -641,24 +650,14 @@ def get_best_block_tips(self, timestamp: Optional[float] = None, *, skip_cache:
         When more than one block is returned, it means that there are multiple best chains and
         you can choose any of them.
         """
-        if timestamp is None and not skip_cache and self._best_block_tips is not None:
```

Comment:
The current method returns even orphan blocks, while the new method returns only the best blockchain. It might have consequences in the block rewards validation. I feel we should put in a different method (e.g., `get_best_blockhash()`) and replace the calls to `get_best_block_tips()`.

Reply:
Makes sense.

Reply:
I don't think we should ever use the new implementation with the previous method name because they are returning different animals. I feel we should create a new method and start using it until `get_best_block_tips()` is not in use anymore and we can delete it. What do you think?

Reply:
Old conversation. It does make sense anymore. Marking as resolved.